index.html

```diff
@@ -51,6 +53,238 @@ <h3>LET’S CONNECT</h3>
           />
         </div>
       </section>
+      <!-- WORK SECTION -->
+      <section id="work">
+        <div class="card-works">
+          <img
+            src="./images/work/Tonic.png"
+            alt="Tonic Image"
+            class="snapshot-portfolio"
+          />
+          <div class="left-block">
+            <div class="title-box">
+              <div class="project-title">Tonic</div>
+              <div class="canopy">
+                <div class="client">CANOPY</div>
+                <div class="counter"></div>
+                <div class="role">Back End Dev</div>
+                <div class="counter"></div>
+                <div class="year">2015</div>
+              </div>
+            </div>
+            <div class="description-box">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </div>
```

Comment:
According to the project requirements, this content should be wrapped inside a `<p>` tag. Kindly fix this.

<img width="345" alt="Screen Shot 2022-05-27 at 17 26 02" src="https://user-images.githubusercontent.com/82426682/170796250-1507c6d6-583e-4dcc-ba90-9954f5cf4d64.png">

You should fix all of your project descriptions, I'm marking just this one for brevity.